docs/entities/index.md

```diff
@@ -0,0 +1,431 @@
+    }
+}
+```
+
+It is also possible to modify damage done to entities that do not belong to you, i.e. those added by Minecraft or other mods, through events. These events contain a lot of code specific to `LivingEntity`s; as such, their documentation resides in the [Damage Events section][damageevents] within the [Living Entities article][livingentity].
+
+### Ticking Entities
+
+Quite often, you will want your entity to do something (e.g. move) every tick. This logic is split across several methods:
+
+- `#tick`: This is the central tick method, and the one you will want to override in 99% of cases.
+    - By default, this forwards to `#baseTick`, however this is overridden by almost every subclass.
+- `#baseTick`: This method handles updating some values common to all entities, including the "on fire" state, freezing from powder snow, the swimming state, and passing through portals. `LivingEntity` additionally handles drowning, in-block damage, and updates to the damage tracker here. Override this method if you want to change or add to that logic.
+    - By default, `Entity#tick` will forward to this method.
+- `#rideTick`: This method is called for passengers of other entities, for example for players riding horses, or any entity that rides another entity due to use of the `/ride` command.
+    - By default, this does some checks and then calls `#tick`. Skeletons and players override this method for special handling of riding entities.
+
+Additionally, the entity has a field called `tickCount`, which is the time, in ticks, that the entity has existed in the level, and a boolean field named `firstTick`, which should be self-explanatory. For example, if you wanted to [spawn a particle][particle] every 5 ticks, you could use the following code:
+
+```java
+@Override
+public void tick() {
+    // Always call super unless you have a good reason not to.
+    super.tick();
+    // Run this code once every 5 ticks, and make sure we spawn the particle on the server.
+    if (this.tickCount % 5 == 0 && !level().isClientSide()) {
+        level().addParticle(...);
+    }
+}
+```
+
+### Picking Entities
+
+_See also [Middle-Clicking][middleclick]._
+
+Picking is the process of selecting the thing that the player is currently looking at, as well as subsequently picking the associated item. The result of middle-clicking, known as the "pick result", can be modified by your entity (be aware that the `Mob` class will select the correct spawn egg for you):
+
+```java
+@Override
+@Nullable
+public ItemStack getPickResult() {
+    // Assumes that MY_CUSTOM_ITEM is a DeferredItem<?>, see the Items article for more information.
+    return new ItemStack(MY_CUSTOM_ITEM.get());
+}
+```
+
+Your entity can also be disabled from picking entirely. The primary use case for this would be multipart entities, such as the ender dragon, where the parent entity has picking disabled, but the parts have it enabled again, for finer hitbox tuning. Picking can be disabled like so:
```

Comment:
I would explicitly mention that, unless in very specific usecases, all entities should be pickable. If not, it should generally just check if the entity isn't removed via `isRemoved` along with whatever other qualifiers are necessary.